ec2.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_instance" "mytftesting" {
+  ami           = "ami-0763cf792771fe1bd"
+  instance_type = "t2.micro"
```

Comment:
Hi @poornima2025 ,
can you please change instance_type from t2.micro to t2.large.